CanonicalLtd/maas-docs
# Test new documentation_builder template on MAAS docs

# Don't merge

Testing how the proposed new documentation_builder template looks on MAAS docs. https://github.com/canonical-webteam/documentation-builder/pull/110

## QA

- Pull code
- `./run`
- Select `update-template` from the version dropdown
- Check the following things:
  - [ ] Overall design looks good (@kwm14)
  - [ ] Header navigation: Search works as before, title shows and is formatted correctly
  - [ ] Sidebar navigation: 3 levels deep items looks good, selected state shows, sticks to top of viewport
  - [ ] Table of contents: Works properly (as before), sticks to top of viewport
  - [ ] Versions: Selecting different versions works, disabled pages work (i.e. versions that don't contain the current page are greyed out)
  - [ ] Media (images) loads in properly

## Screenshot

### Before

![docs maas io_2 4_en_](https://user-images.githubusercontent.com/25733845/41033626-2c54f002-697f-11e8-903b-b3865074aeb1.png)

### After

![0 0 0 0_8203_update-template_en_ 1](https://user-images.githubusercontent.com/25733845/41285428-05241a8a-6e34-11e8-8468-c71861fcfacb.png)
